client/src/components/Navbar/Navbar.js

```diff
@@ -24,49 +35,52 @@ function Navbar() {
 
                         <Link to ="/" id="navLink">
                             <hr className="d-md-none" />
-                            <li className={(active==='Home')?"nav-item mx-3 px-3 active":"nav-item mx-3 px-3"} 
-                            onClick={() => setActive('Home')}>
+                            <li className={(active==='home')?"nav-item mx-3 px-3 active":"nav-item mx-3 px-3"} 
```

Comment:
I would like to suggest that we should keep the styling or indenting part separate from our react code because as we proceed, at a later stage it could interfere in the responsiveness of our webpage, and most importantly once separated it will be easy for debugging.
By separating here I mean adding margin or padding in .css file rather than using mx or px.
It is only a suggestion though :)

Reply:
On point @hariom1625 💯 This should be changed for sure. I used mx px in #12 so, consider fixing this in your next contribution. Or if @RishavMz is willing to take this up.

Reply:
As @RishavMz is currently working on it so his call :-)

Reply:
Yes, would be better if you style this @hariom1625. I face a bit trouble with margins and paddings😅.

Reply:
Yes sure @RishavMz :-)